tests/lib/snaps/test-snapd-physical-memory-observe/bin/head-mem

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+head /dev/mem
```

Comment:
(nitpick) This will print the first 10 lines, how much data is displayed will depend on the (random) distribution of \n in bytes in /dev/mem. I think it would be slightly more robust to use: `head -c10 /dev/mem|od -x` or similar, i.e. only display the first 10 char and show as hex (just just `od -N10 -x /dev/mem`).

Reply:
Ok, scratch that comment about using `od` - that is not available in the default template.

Reply:
You may run into problems with non printable characters. Why not `dd if=/dev/mem of=/dev/null bs=1024 count=1` instead?